game/WiiU3DSMiitomo_miidatafile.ksy

```diff
@@ -0,0 +1,178 @@
+meta:
+  id: wiiu3dsmiitomo_miidatafile

```

Comment:
The `id` is okay now, but the file name still has uppercase letters - it should also be all lowercase like the `id`.

If you're on Windows or Mac, it's possible that you've already renamed the file to lowercase and Git simply didn't detect the change (this can happen because Windows and Mac use case-insensitive filesystems by default). In that case you might have to use GitHub's web interface to rename the file to lowercase.